_episodes/02-file_parsing.md

```diff
@@ -196,10 +204,17 @@ print(words)
 
 From this `print` statement, we now see that we have a list called words, where we have split `energy_line`.  The energy is actually the fourth element of this list, so we can now save it as a new variable.
 
-```
+```python
 energy = words[3]
 print(energy)
 ```
+
+We also recogize that "energy" is the last element of the list. Therefore, an alternative way to assign `energy` is:
```

Comment:
Can you add this as a callout? (yellow boxed information) The callout would be about other ways to access list elements (negative indexing)

The syntax would be

> ## Title
> Text
> Text
{: .callout}

Reply:
I have switched this block to callout format! I hope I did it correctly. I am not familiar with carpentries format!

Reply:
Thank you! I have previewed it locally and this appears to be the correct format.